cmake/sip_configure.py

```diff
@@ -61,6 +61,12 @@ def get_sip_dir_flags(config):
         # sipconfig.Configuration does not have a pyqt_sip_dir or pyqt_sip_flags AttributeError
         sip_flags = QtCore.PYQT_CONFIGURATION['sip_flags']
 
+        # sip5 installs here by default
+        default_sip_dir = os.path.join(sipconfig._pkg_config['default_mod_dir'], 'PyQt5', 'bindings')
```

Comment:
I launched an archlinux container and [confirmed this is the right location](https://archlinux.org/packages/extra/x86_64/python-pyqt5/).

```python3
>>> sipconfig._pkg_config['default_mod_dir']
'/usr/lib/python3.9/site-packages'
>>> sipconfig._pkg_config['default_sip_dir']
'/usr/share/sip'
```

It does seem strange since `default_sip_dir` seems like it should be the right spot.

> default_sip_dir
>
>    The name of the base directory where the .sip files for SIP generated modules should be installed by default. A sub-directory with the same name as the module should be created and its .sip files should be installed in the sub-directory. The .sip files only need to be installed if you might want to build other bindings based on them.

Is it possible this is an archlinux specific packaging bug? Maybe [introduced here](https://github.com/archlinux/svntogit-packages/commit/bd544723e6262bc978d4a7680eedeef90186dc41#diff-37538beb61ff63edebbf735dfcf39e5d732f49183d6beb097169d971875ca422L48)? If so, I would be happy to merge this addition so long as the comment says it's a workaround for packaging on archlinux.

Reply:
thx for looking into this! Yeah I'll update the comment to be more reflective of the situation.

Reply:
Renamed the comment and commit to mention the change is due to archlinux specifics.

Reply:
There is nothing archlinux specific here. You are looking at the sip 4 install dir, this change is for sip >= 5 (latest version is 6.x)